components/search/autocomplete.js

```diff
@@ -26,13 +26,39 @@ class AutoComplete extends Component {
   }
 
   componentDidMount() {
+    document.addEventListener('keyup', this.onKeyUp.bind(this), false)
+
     if (this.props.router.query.query) {
       this.setState({
         value: decodeURIComponent(this.props.router.query.query) || ''
       })
     }
   }
 
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.onKeyUp.bind(this), false)
+  }
+
+  onKeyUp(event) {
+      switch (event.keyCode) {
+          case 55:
```

Comment:
I believe the code for <kbd>/</kbd> is `47`
```suggestion
          case 47:
```

For me and @msweeneydev, this code is for the <kbd>7</kbd> key.

Reply:
Oh yeah, I'm using an European keyboard and my <kbd>/</kbd> is mapped to <kbd>shift</kbd> + <kbd>7</kbd>, must have copied the wrong code 😅. Fixing now!